### PR TITLE
Update hotfix docs

### PR DESCRIPTION
**Pull request by sanderegg**

### CODE REVIEW

File: docs/releasing-workflow-instructions.md [hunk 1]
```diff
@@ -132,13 +133,22 @@ A bug was found in version 1.2.0 of the simcore stack. The team decides to fix i
...
   cd osparc-simcore
+  # let's checkout the release with the issue, typically a release tag such as v1.4.5
   git checkout VERSION_TAG_FOR_HOTFIXING
-  # make the fix through usual PR process
+  # create the hotfix branch, the name must follow the hotfix_v* convention, what lies after v is free
+  git checkout -b hotfix_v1_4_x
+  # develop the fix here, git commit, git push, have someone review your code
+  # git commit -m "this is my awsome fix for this problematic issue"
+  # git push --set-upstream origin/hotfix_v1_4_x
+  # wait until the CI completed the its run (going through ALL the tests and generating the docker images)
```
Comment (odeimaiz) [inline docs/releasing-workflow-instructions.md:143]:
```suggestion
  # wait until the CI completes its run (going through ALL the tests and generating the docker images)
```

File: docs/releasing-workflow-instructions.md [hunk 2]
```diff
@@ -132,13 +133,22 @@ A bug was found in version 1.2.0 of the simcore stack. The team decides to fix i
   ```bash
   git clone https://github.com/ITISFoundation/osparc-simcore.git
   cd osparc-simcore
+  # let's checkout the release with the issue, typically a release tag such as v1.4.5
   git checkout VERSION_TAG_FOR_HOTFIXING
-  # make the fix through usual PR process
+  # create the hotfix branch, the name must follow the hotfix_v* convention, what lies after v is free
+  git checkout -b hotfix_v1_4_x
+  # develop the fix here, git commit, git push, have someone review your code
+  # git commit -m "this is my awsome fix for this problematic issue"
```
Comment (odeimaiz) [inline docs/releasing-workflow-instructions.md:141]:
```suggestion
  # git commit -m "this is my awesome fix for this problematic issue"
```

**Review by odeimaiz — APPROVED** [review]
awesome!

**Review by pcrespov — APPROVED** [review]

**Comment by pcrespov** [comment]
@sanderegg It would be convenient to have something that skips CI when we modify only docs: chekc this: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/

**Review by GitHK — APPROVED** [review]

**Pull request merged**